run.php

```diff
@@ -25,61 +26,81 @@
 $REPOS_WITH_PRS_CREATED = [];
 $OUT = null;
 
+// options
+$optionCmsMajor = [
```

Comment:
This is refactoring to do to prevent code duplication because the existing "update" command and the new "labels" command share many options